scripts/global_state.py

```diff
@@ -83,6 +83,7 @@ def unified_preprocessor(preprocessor_name: str, *args, **kwargs):
     "shuffle": shuffle,
     "tile_resample": tile_resample,
     "invert": invert,
+    "greyscale": greyscale,
```

Comment:
Add an entry to `preprocessor_aliases` to map `grayscale` alternative spelling

Reply:
You can add an entry to `preprocessor_sliders_config`
```
    "greyscale": [
        {
            "name": flag_preprocessor_resolution,
            "value": 512,
            "min": 64,
            "max": 2048,
        }
    ],
```

and to `preprocessor_filters`
```
    "Greyscale": "greyscale",
```